src/components/BooksPage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+const bookList = [
+  {
+    booktitle: ' Pride and Prejudice',
+    author: ' Jane Austen',
+    Category: ' Romance',
+    id: '1',
+  },
+  {
+    booktitle: ' The Great Gatsby',
+    author: ' F. Scott Fitzgerald',
+    Category: ' Fiction',
+    id: '2',
+  },
+];
+const BooksPage = () => (
+  <div className="books">
+    <ul className="books-list-section">
+      {bookList.map((book) => (
+        <li key={book.id}>
+          <p>
+            Book:
+            {book.booktitle}
+          </p>
+          <p>
+            Author:
+            {book.author}
+          </p>
+          <p>
+            Category:
+            {book.Category}
+          </p>
+          <button type="button">Remove</button>
+        </li>
+      ))}
```

Comment:
- [ ] Your component for rendering books, in this case, `BookPage.js`, should list the books using a `Book` component. That means you should create a new `Book` component responsible for a single book and extract this mapping functionality into it.
  
  - [ ] The new Book component should take in the book data as a prop and render the book title and author (or category) as you have here
  - [ ] This new Book component should also have a placeholder Remove button as you have here. The button does not have to work yet.

Reply:
Done